### PR TITLE
Task 2 person class

### PR DESCRIPTION
**Pull request by 3433825**
Implement Person, Student, Teacher classes

### CODE REVIEW

**Comment by denyskulykov** [comment]
You must use only English 
commit message - "уточнение задачи"

File: task_massive_similar.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from difflib import SequenceMatcher
+from string import punctuation
+# import re
+
+words = []
```
Comment (denyskulykov) [inline task_massive_similar.py:5]: move it to func

**Comment by denyskulykov** [comment]
Need to update files

**Comment by 3433825** [comment]
develop already contains these changes

**Pull request closed without merging**